mlx/robot2rst.mako

```diff
@@ -32,54 +47,33 @@ ${'='*len(suite)}
     :depth: 2
     :local:
 
---------
```

Comment:
I would also like to parse/display settings to keep this somewhere. It soubds like we are making robotdoc replacement, but i dont think we should throw away these

Reply:
We decided to stop extracting them and only extract test cases with their tags and documentation